cmake/radium_setup_functions.cmake

```diff
@@ -429,9 +429,10 @@ function(installTargetResources)
         set(ARGS_PREFIX "")
     endif ()
     if (ARGS_FILES)
+        message("[${ARGS_FILES}]")
```

Comment:
Do we really need this peint ? Or was it for debugging ?

Reply:
I really don't know what you're talking about, like I was debugging by printing messsage ? :smile:

Reply:
:D